lib/europeana/page.rb

```diff
@@ -55,8 +55,18 @@ def credit_elements
       }
     end
 
+    def credit_image
+      return @credit_image if instance_variable_defined?(:@credit_image)
```

Comment:
Will this method ever be called more than once per page render? If not, consider removing the memoization.

Reply:
it wouldn't be now, but I think in future especially when implementing automatic foyer cards, this has potential to be called multiple times for an exhibition.